docs/reference/settings/ilm-settings.asciidoc

```diff
@@ -23,6 +23,17 @@ policy that contains a rollover action. When the index rolls over, the alias is
 updated to reflect that the index is no longer the write index. For more
 information about rollover, see <<using-policies-rollover>>.
 
+`index.lifecycle.rollover.skip_rolled`:: (Expert Setting)
```

Comment:
Is there a better suggestion for naming this setting? I also went ahead and made it `true` by default. What do you think?

Reply:
`allow_multiple_rollovers` maybe? I don't love that either.
`skip_rollover_if_already_rolled`? More verbose, but that might be okay.

I'm honestly not sure why someone would want to change this setting in the first place. It's probably good to have a setting for it just in case, but I can't think of any actual use case for it.

Reply:
I think this setting is a little strange, if a user were to set it to false they would essentially be rolling over an index that was no longer the write index, and in the case that ILM is configured like it usually is (the policy is applied to every index), it will lead to unintended behavior.

I think we should remove this setting, since to me this feels a little like having a setting to re-introduce buggy behavior.